docs/3_ObjectRecognition.md

```diff
@@ -0,0 +1,486 @@
+# Object recognition
+
```

Comment:
# Besoin d'une section Objectif : 

For the project to work we need the Strandbeests to identify people in distress. Our Strandbeests have cameras and Raspberry Pis, so we can use Object Recognition.  This is a challenge because we have limited compute power.